core/src/main/scala/scala/collection/parallel/Tasks.scala

```diff
@@ -289,7 +289,7 @@ trait ForkJoinTasks extends Tasks with HavingForkJoinPool {
 }
 
 object ForkJoinTasks {
-  lazy val defaultForkJoinPool: ForkJoinPool = new ForkJoinPool()
+  lazy val defaultForkJoinPool: ForkJoinPool = ForkJoinPool.commonPool()
```

Comment:
@axel22 Hey Aleks, do you remember why this wasn't like that before? (Due to having to support Java < 8?)

Reply:
I think the reason might have been supporting JDK 6 even, but I'm not really sure.

Reply:
@axel22 Makes sense. Do you foresee any issues moving to the commonPool()?

Reply:
No, I can't see any issues with that at the moment.

Reply:
@axel22 True, at least there will be one less pool to manage :)